icons/poi_school.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
+ <g style="display:inline">
+  <path style="display:inline;fill:#f9f5f0;fill-opacity:1;stroke:#fff;stroke-width:0;stroke-linecap:round;paint-order:stroke markers fill" d="M 9,0 A 2.0002,2.0002 0 0 0 8.2304688,0.15429688 l -6,2.50000002 a 2.0002,2.0002 0 0 0 0,3.6914062 L 6.1992188,8 H 6 a 2.0002,2.0002 0 0 0 -2,2 v 8 a 2.0002,2.0002 0 0 0 2,2 h 8 a 2.0002,2.0002 0 0 0 2,-2 V 10 A 2.0002,2.0002 0 0 0 14,8 H 13 V 2 A 2.0002,2.0002 0 0 0 11,0 Z"/>
+  <path style="display:inline;fill:#003f87;fill-opacity:1;stroke:#fff;stroke-width:0;stroke-linecap:round;paint-order:stroke markers fill" d="M 9.0000001,1.9999999 3,4.5 9.0000001,6.9999993 Z M 6,10 v 8 h 8 V 10 Z M 9,2 v 8 h 2 V 2 Z"/>
```

Comment:
Bring the top of the flagpole to a sharper point. This may technically distort the pennant a little, but as an optical illusion, the flagpole won’t look curved at actual size.

Reply:
I'm not sure I understand what you mean, something like this?
![poi_school_1pxknockout_point](https://user-images.githubusercontent.com/58491489/212502011-3f3ecfaa-509d-497a-8c77-62886b3b2ab7.svg)

Reply:
Maybe easier to see in inkscape
<img width="399" alt="Screen Shot 2023-01-14 at 3 48 17 PM" src="https://user-images.githubusercontent.com/58491489/212502040-7d3bece5-823e-49a0-848b-1144ba4990ff.png">

Reply:
That’s better, but we could even do away with the bevel:

![sharp](https://user-images.githubusercontent.com/1231218/212502312-5020c14c-deb8-48b4-8909-8b4501246a73.svg)

Reply:
Ah understood, that is kind of nice I think
<img width="294" alt="Screen Shot 2023-01-14 at 4 12 02 PM" src="https://user-images.githubusercontent.com/58491489/212502667-f941ef5e-3ca4-4c4e-935b-afb6c079c96a.png">